### PR TITLE
[bugfix] Fix error message when a sane Tmod installation cannot be found

### LINKED ISSUE #1408: Redundant information in error message when a sane Tmod installation cannot be found
```
$ ./bin/reframe -C config/cscs.py --system=dom --show-config=systems
./bin/reframe: could not connect to Graylog server at 'graylog-server:12345': [Errno 8] nodename nor servname provided, or not known
./bin/reframe: could not initialize the graylog handler; ignoring ...
./bin/reframe: failed to initialize runtime: could not find a sane TMod installation: [Errno 2] No such file or directory: 'modulecmd': 'modulecmd': [Errno 2] No such file or directory: 'modulecmd': 'modulecmd'
```

### PR DESCRIPTION
**Pull request by ekouts**
Fixes #1408 .

### CODE REVIEW

**Comment by ekouts** [comment]
@jenkins-cscs retry all

**Review by victorusu — APPROVED** [review]
`lgtm`

File: reframe/core/modules.py [hunk 1]
```diff
@@ -421,7 +421,7 @@ def __init__(self):
             completed = os_ext.run_command('modulecmd -V')
         except OSError as e:
             raise ConfigError(
-                'could not find a sane TMod installation: %s' % e) from e
+                'could not find a sane TMod installation:') from e
```
Comment (vkarak) [inline reframe/core/modules.py:424]: Remove the `:` at the end. The exception chaining mechanism will insert it.

**Review by vkarak — CHANGES_REQUESTED** [review]

File: reframe/core/modules.py (hunk 1 above)
Comment (ekouts) [reply reframe/core/modules.py]: Ah yes sorry, you are right! I though I had deleted it

**Review by vkarak — APPROVED** [review]

**Pull request merged**